### ISSUE TITLE
only works when using localhost and not on any server

### ISSUE THREAD

**Issue by Nidheeshv0312** [muaz-khan/RecordRTC#771]
Uncaught TypeError: navigator.getUserMedia is not a function
    onStartRecording http://192.168.1.199:8000/js/index.js:104
    onload http://192.168.1.199:8000/js/index.js:119
    EventHandlerNonNull* http://192.168.1.199:8000/js/index.js:116
index.js:104:15

**Comment by createdbymahmood** [comment]
I think you need to be in an HTTPS context to make that work

**Comment by isaiahstjohn** [comment]
Yes, you need to be in a secure context. See: https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#security
